admin-core/src/main/kotlin/com/epam/drill/admin/DrillApplication.kt

```diff
@@ -169,6 +184,14 @@ private fun Application.installPlugins() {
     install(RoleBasedAuthorization)
 }
 
+private fun StatusPages.Configuration.defaultStatusPages() {
+    exception<Throwable> { cause ->
+        logger.error(cause) { "Build application finished with exception" }
```

Comment:
I don't get what `"Build application"` means
If I understand correctly, this log is always capturing request handling exception. If that is correct, I suggest to write `"Failed to process the request"` and make sure to add both request details and exception details to trace (request url, payload, headers, cookies, etc + exception message and stacktrace)